### PR TITLE
PX4Accelerometer/PX4Gyroscope: integrated data use simple rotate_3f to avoid unnecessary floating point operations

### PR DESCRIPTION
**Pull request by dagar**
Possible fix for (part of) https://github.com/PX4/Firmware/issues/14735.

Screenshot from 2020-04-24 10-02-20

### CODE REVIEW

**Comment by dagar** [comment]
Screenshot from 2020-04-24 11-55-48

**Comment by dagar** [comment]
It's a bit hard to see because the raw sensor data was captured at a low rate, but if we filter both the raw data and integrated data of the same axis side-by-side you can see the difference.

Screenshot from 2020-04-24 12-02-40


This is not present in the y or z axis.

Screenshot from 2020-04-24 12-04-15

Screenshot from 2020-04-24 12-05-48

**Comment by dagar** [comment]
This can be merged whether it fixes any potential numerical problem or not. The simple `rotate_3f()` helper is cheaper than the rotation matrix.

File: src/lib/drivers/accelerometer/PX4Accelerometer.cpp [hunk 1]
```diff
@@ -266,16 +265,14 @@ void PX4Accelerometer::updateFIFO(const FIFOSample &sample)
...
-			// Apply rotation and scale
-			// integrated in microseconds, convert to seconds
-			const Vector3f delta_velocity_uncalibrated{_rotation_dcm *_integration_raw * _scale};
+			// Apply rotation (before scaling)
+			rotate_3f(_rotation, _integration_raw(0), _integration_raw(1), _integration_raw(2));
 
-			// scale calibration offset to number of samples
-			const Vector3f offset{_calibration_offset * _integrator_fifo_samples};
+			// integrated in microseconds, convert to seconds
+			const Vector3f delta_velocity_uncalibrated{_integration_raw * 1e-6f * dt * _scale};
```
Comment (bkueng) [inline src/lib/drivers/accelerometer/PX4Accelerometer.cpp:272]: You can reduce the number of operations here if you add brackets:
```
delta_velocity_uncalibrated{_integration_raw * (1e-6f * dt * _scale)};
```

**Pull request merged**